components/script/dom/canvasrenderingcontext2d.rs

```diff
@@ -130,9 +130,10 @@ impl CanvasRenderingContext2D {
                          -> CanvasRenderingContext2D {
         debug!("Creating new canvas rendering context.");
         let (sender, receiver) = ipc::channel().unwrap();
+        let canvas_channel = ipc::channel::<CanvasMsg>().unwrap();
```

Comment:
We should be able to do `let (sender, ipc_renderer) = ipc::channel::<CanvasMsg>().unwrap()` and remove the `let ipc_renderer` and `let (sender, receiver)` lines entirely.